lib/widgets/mapPoint_widget.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:gra_terenowa/controller/mapData_controller.dart';
+import 'package:gra_terenowa/controller/mapTransfer_controller.dart';
+import 'package:gra_terenowa/model/mapDatabase.dart';
+import 'package:gra_terenowa/statics/colors.dart';
+import 'package:gra_terenowa/statics/constants.dart';
+
+class MapPoint extends StatelessWidget {
+  MapPoint({
+    Key? key,
+    required this.mapDataController,
+    required this.mapTransformController,
+    required this.mapItemIndex,
+    required this.scrollController,
+  }) : super(key: key);
+
+  final MapDataController mapDataController;
+  final MapTransformController mapTransformController;
+  final int mapItemIndex;
+  final ScrollController scrollController;
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
It might be good to show which item is activated on the bottom bar, maybe by highlightig the card. What do you think?

Reply:
I like the idea and implemented it by highlighting the info icon on the card with the same color as the map icon (exact choice of color might need some testing).